src/test/java/io/jenkins/plugins/checks/github/GitHubChecksPublisherITest.java

```diff
@@ -159,56 +159,66 @@ public static Collection<Object[]> contextBuilders() {
      */
     @Test
     public void shouldPublishGitHubCheckRunCorrectly() {
-        ChecksDetails details = new ChecksDetailsBuilder()
-                .withName("Jenkins")
-                .withStatus(ChecksStatus.COMPLETED)
-                .withDetailsURL("https://ci.jenkins.io")
-                .withStartedAt(LocalDateTime.ofEpochSecond(999_999, 0, ZoneOffset.UTC))
-                .withCompletedAt(LocalDateTime.ofEpochSecond(999_999, 0, ZoneOffset.UTC))
-                .withConclusion(ChecksConclusion.SUCCESS)
-                .withOutput(new ChecksOutputBuilder()
-                        .withTitle("Jenkins Check")
-                        .withSummary("# A Successful Build")
-                        .withText("## 0 Failures")
-                        .withAnnotations(Arrays.asList(
-                                new ChecksAnnotationBuilder()
-                                        .withPath("Jenkinsfile")
-                                        .withLine(1)
-                                        .withAnnotationLevel(ChecksAnnotationLevel.NOTICE)
-                                        .withMessage("say hello to Jenkins")
-                                        .withStartColumn(0)
-                                        .withEndColumn(20)
-                                        .withTitle("Hello Jenkins")
-                                        .withRawDetails("a simple echo command")
-                                        .build(),
-                                new ChecksAnnotationBuilder()
-                                        .withPath("Jenkinsfile")
-                                        .withLine(2)
-                                        .withAnnotationLevel(ChecksAnnotationLevel.WARNING)
-                                        .withMessage("say hello to GitHub Checks API")
-                                        .withStartColumn(0)
-                                        .withEndColumn(30)
-                                        .withTitle("Hello GitHub Checks API")
-                                        .withRawDetails("a simple echo command")
-                                        .build()))
-                        .withImages(Collections.singletonList(
-                                new ChecksImage("Jenkins",
-                                        "https://ci.jenkins.io/static/cd5757a8/images/jenkins-header-logo-v2.svg",
-                                        "Jenkins Symbol")))
-                        .build())
-                .withActions(Collections.singletonList(
-                        new ChecksAction("re-run", "re-run Jenkins build", "#0")))
-                .build();
-
-        new GitHubChecksPublisher(contextBuilder.apply(this),
-                new PluginLogger(j.createTaskListener().getLogger(), "GitHub Checks"),
-                wireMockRule.baseUrl())
-                .publish(details);
+        try (var credentialsMatchers = mockCredentialsMatchers()) {
```

Comment:
Introduced `try-with-resource` for static mock auto-close. The body is indented without any change.